src/reports_api/models/project.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy import Column, ForeignKey, Integer, Boolean, String, Float
+from sqlalchemy.orm import relationship
+
+from .code_table import CodeTable
+from .db import db
+
+
+class Project(db.Model, CodeTable):
+    """Model class for Project."""
+
+    __tablename__ = 'projects'
+
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    description = Column(String())
+    location = Column(String())
+    capital_investment = Column(Float())
```

Comment:
Is it by default nullable=True, if not, I think it should be true. I couldn't find this value is being set in the project creation screen in the access based application of theirs.

Reply:
Tested using postman and it worked similar to nullable=True